common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2892,3 +2894,26 @@ func (d *cassandraPersistence) RangeDeleteReplicationTaskFromDLQ(
 	}
 	return nil
 }
+
+func (d *cassandraPersistence) CreateFailoverMarkerTasks(
+	request *p.CreateFailoverMarkersRequest,
+) error {
+
+	batch := d.session.NewBatch(gocql.LoggedBatch)
+	for _, task := range request.Markers {
+		t := []p.Task{task}
```

Comment:
I think I am missing some context here. `createReplicationTasks` can take a list of tasks, why do we need to invoke it for only one task at a time?

Reply:
The tasks have different domain ID so it has to update one at a time.